### PR TITLE
[NEW][9.0] project_task_default_stage: Default stages for new projects.

### PR DESCRIPTION
**Pull request by sergio-teruel**
Wiith this module we recovery default stages for new project as in v8.0
@rafaelbn, @dreispt, @pedrobaeza what you think about this?

### CODE REVIEW

File: project_case_default/README.rst [hunk 1]
```diff
@@ -0,0 +1,75 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+====================
+Project Case Default
```
Comment (elicoidal) [inline project_case_default/README.rst]: Project Default Stage would be better
Module name: project_default_stage

File: project_case_default/README.rst [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+====================
+
+This module restores the ``case_default`` fields to Project Tasks and project
+stage tab, removed in Odoo 9.0.
+
+Configuration
+=============
+
+After installing this module you can assign stage tasks into project stage tab
+and you can check the default stages for new projects as was done in odoo 8.0
```
Comment (elicoidal) [inline project_case_default/README.rst]: as previously done in Odoo 8.0.

File: project_case_default/README.rst [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/140/9.0
+
+For further information, please visit:
```
Comment (elicoidal) [inline project_case_default/README.rst]: remove

File: project_case_default/README.rst [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/project-service/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/project-service/issues/new?body=module:%20project_case_default%0Aversion:%209.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+
+License
```
Comment (elicoidal) [inline project_case_default/README.rst]: remove

File: project_case_default/README.rst [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (elicoidal) [inline project_case_default/README.rst]: https

File: project_case_default/README.rst (hunk 1 above)
Comment (sergio-teruel) [inline project_case_default/README.rst]: Thanks for your review..
I think that this name can create confusion between project stage and task stages. The default stage is not for project, is for task. 
What do you think about _project_task_default_stage_?

Comment (elicoidal) [inline project_case_default/README.rst]: OK for project_task_default_stage

**Comment by sergio-teruel** [comment]
@elicoidal Changes done. Thanks.

**Comment by dreispt** [comment]
AFAIK the default stage is the first  in the Project's stage list.
Can you share a use case this is not enough?

**Comment by sergio-teruel** [comment]
In Odoo 9.0 you can create new task stages and assign what project uses this task stages, i think that this is not very comfortable. I think that is more usefull in v 8.0, you can defines task stages and check if you want use this stage by default in your new projects.
In Odoo 9.0 when creates a new project, you have to create new task stages or assign this new project to the task stages.

**Comment by dreispt** [comment]
@sergio-incaser OK, I see now. I misunderstood the purpose.

**Comment by sergio-teruel** [comment]
@falinwa This module resolve https://github.com/odoo/odoo/issues/9030. Can you review please?

**Comment by aongunner** [comment]
This module can help me. Thank a lot.

**Comment by dreispt** [comment]
LGTM :+1:

**Pull request merged**